utils/test_harness/sysman/src/test_harness_sysman_power.cpp

```diff
@@ -85,30 +99,39 @@ void compare_power_descriptor_structures(
             secondDescriptor.limitValueLocked);
   EXPECT_EQ(firstDescriptor.limit, secondDescriptor.limit);
 }
-void get_power_limits(zes_pwr_handle_t pPowerHandle,
+ze_result_t get_power_limits(zes_pwr_handle_t pPowerHandle,
                       zes_power_sustained_limit_t *pSustained,
                       zes_power_burst_limit_t *pBurst,
                       zes_power_peak_limit_t *pPeak) {
-  EXPECT_EQ(ZE_RESULT_SUCCESS,
-            zesPowerGetLimits(pPowerHandle, pSustained, pBurst, pPeak));
+  ze_result_t status =
+      zesPowerGetLimits(pPowerHandle, pSustained, pBurst, pPeak);
+  if (status != ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
+    EXPECT_EQ(ZE_RESULT_SUCCESS, status);
+  }
+  return status;  
 }
-void set_power_limits(zes_pwr_handle_t pPowerHandle,
+ze_result_t set_power_limits(zes_pwr_handle_t pPowerHandle,
                       zes_power_sustained_limit_t *pSustained,
                       zes_power_burst_limit_t *pBurst,
                       zes_power_peak_limit_t *pPeak) {
-  EXPECT_EQ(ZE_RESULT_SUCCESS,
-            zesPowerSetLimits(pPowerHandle, pSustained, pBurst, pPeak));
+  ze_result_t status =
+      zesPowerSetLimits(pPowerHandle, pSustained, pBurst, pPeak);
+  if (status != ZE_RESULT_ERROR_UNSUPPORTED_FEATURE) {
```

Comment:
Please handle other errors (other than SUCCESS or UNSUPPORTED)

Reply:
I did not get the request. We are not handling just bypassing the current check `EXPECT_EQ(ZE_RESULT_SUCCESS, status);` when the status is `ZE_RESULT_ERROR_UNSUPPORTED_FEATURE`. This is done since `ZE_RESULT_ERROR_UNSUPPORTED_FEATURE` is the only acceptable case when we should not error out.

Reply:
What happens if error other than UNSUPPORTED is encountered

Reply:
removed these from harness